package.json

```diff
@@ -5,7 +5,7 @@
   "main": "main.js",
   "productName": "Jitsi Meet",
   "scripts": {
-    "start": "electron .",
+    "start": "electron --enable-transparent-visuals --disable-gpu .",
```

Comment:
Temporarily added a gpu disable command for linux (I use Ubuntu 16.04). Not tested yet for other OS